.bazelrc

```diff
@@ -1,5 +1,10 @@
 # Envoy specific Bazel build/test options.
 
+# Bazel doesn't need less than 200MB of memory based on memroy profiling:
```

Comment:
spelling: memroy

Reply:
Also, this sentence is hard to parse. It reads to me as "bazel does need more than 200MB", but I'm not sure that's what you meant.

Reply:
thanks! yes that what I meant 🤦‍♂️